src/SuperDumpService/Services/SearchService.cs

```diff
@@ -65,7 +68,8 @@ public class SearchService {
 				new RetentionViewModel(
 					dumpMetainfo,
 					dumpRepo.IsPrimaryDumpAvailable(dumpMetainfo.Id),
-					TimeSpan.FromDays(settings.WarnBeforeDeletionInDays)));
+					TimeSpan.FromDays(settings.WarnBeforeDeletionInDays),
+					settings.UseJiraIntegration && jiraIssueRepository.IsPopulated && jiraIssueRepository.HasBundleOpenIssues(dumpMetainfo.BundleId)));
```

Comment:
would you mind adding a performance test here? we've recently been burned by a suboptimal method being called in here. be aware that ToDumpViewModel is called for every dump in the repo when the listview is created. think about 100k+ dumps.

use the benchmarkdotnet project. create a new test that benchmarks the SearchService, especially the simple search.

Reply:
Thank you for the feedback. I added a benchmark that compares the performance of the simple search with and without Jira integration enabled.

According to the benchmark, the difference is about 70 milliseconds for 100 000 sample dumps on my machine.